docker/kubernetes/README.md

```diff
@@ -0,0 +1,234 @@
+* Starting Gameroom
+* Creating users and logging in to the web app
+* Playing Tic-tac-toe
+
+## Prerequisites
+
+Installation and configuration of Minikube is beyond the scope of this
+walkthrough. For specific instructions see the
+[Minikube installation documentation](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+## Deploy Gameroom
+
+### Step 1: Generate Keys
+
+Run a job to generate user keys:
+
+`$ kubectl apply -f https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/create-gameroom-keys.yaml`
+
+View the output from the job:
+
+`$ jobpod=$(kubectl get pods --selector=job-name=gameroom-keys --output=jsonpath='{.items[*].metadata.name}')`
+`$ kubectl logs $jobpod`
+
+The output will be similar to this:
+
+```
+alice.priv: 36a7e8cc7c81fdc7c7278aa157a732c283b9ba0e2ee4ec363992d3d96bad8207
+alice.pub: 031bc8f3a326766ce628295ad6f01c49fd416306a0cf55cca42a6b87e3eccc3c8f
+bob.priv: 8013752e26576e27e31105f6cf0f4537dc8a381a5b347a09959292cb4b17b388
+bob.pub: 029d011c4c4f058b2b5a4cdff1dbe1749809cb121eb45dc766292a26d3a9b84fcd
+```
+
+Copy these values to a scratchpad or keep them available in your terminal
+because you'll be using them a few times in the walkthrough.
+
+### Step 2: Update the KeyRegistry Template
+
+Download the KeyRegistry template: [key-registry.yaml](https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/key-registry.yaml).
+
+Edit `key-registry.yaml` to add the public key values created above.
+
+`$ vim key-registry.yaml`
+
+Replace the lines that say `alice.pub` with the corresponding key data generated
+from the job run in the previous step.
+
+Next, do the same with the `bob.pub` lines.
+
+Make sure that the YAML is correctly formatted. It should look similar to this:
+
+```
+---
+031bc8f3a326766ce628295ad6f01c49fd416306a0cf55cca42a6b87e3eccc3c8f:
+  public_key: 031bc8f3a326766ce628295ad6f01c49fd416306a0cf55cca42a6b87e3eccc3c8f
+  associated_node_id: acme
+  metadata:
+    gameroom/first-name: alice
+    gameroom/organization: Acme
+029d011c4c4f058b2b5a4cdff1dbe1749809cb121eb45dc766292a26d3a9b84fcd:
+  public_key: 029d011c4c4f058b2b5a4cdff1dbe1749809cb121eb45dc766292a26d3a9b84fcd
+  associated_node_id: bubba
+  metadata:
+    gameroom/first-name: bob
+    gameroom/organization: Bubba Bakery
+```
+
+### Step 3: Create ConfigMaps for KeyRegistry and NodeRegistry
+
+`$ kubectl create configmap key-registry --from-file key-registry.yaml`
+
+Download the NodeRegistry file: [node-registry.yaml](https://raw.githubusercontent.com/Cargill/splinter/master/docker/kubernetes/node-registry.yaml).
+
+`$ kubectl create configmap node-registry --from-file node-registry.yaml`
+
+Verify the ConfigMaps were created:
+
+```
+$ kubectl get cm
+NAME            DATA   AGE
+key-registry    1      30s
+node-registry   1      2m43s
+```
+
+You can inspect the values of the ConfigMaps by running
+`kubectl describe cm <configmapname>`. For example:
+
+```
+$ kubectl describe cm key-registry
```

Comment:
This command describes the key-registry config map, but the data for the node-registry map is shown in the example.

Reply:
@jsmitchell oops. Fixed.